docs/LDAP.rst

```diff
@@ -16,6 +16,10 @@ LDAP Integration
 
```

Comment:
We should remove "The fact that an identity which is created via eXo Platform UI can be saved into the LDAP directory differentiates this integration with SSO use cases." since it is not true anymore.